docs/ProjectManagement-AgileAdaptation.md

```diff
@@ -9,42 +9,59 @@ Our project management process is a form of Agile that has been adapted (and con
 
 Each project has the following roles:
 
-* Project lead: responsible for all things. We take turns being project lead. The project lead runs the [project meetings](#meetings)
-* Developer or designer: anyone else who’s working on the project.
+* Product owner: a member of the client team. Prioritizes product scope and features. Works with development team to fully define scope of upcoming work, reviews and signs off on completed features in a timely manner.
+* Project lead: a member of the technical team. Responsible for all things. We take turns being project lead. The project lead runs the [project meetings](#meetings)
+* Developer or designer: anyone else on the technical team who’s working on the project.
+* Delivery manager: keeps an eye on schedule, scope and workflow. Liase with Product owner on budget, schedule and scope decisions.
 * Project advisor: someone who’s not actively working on the project but keeps an eye on things and helps out where needed.
```

Comment:
@pauldowman we don't formally assign this role anymore.  Should we re-initiate it or remove?